src/Entities/Schema.php

```diff
@@ -177,9 +177,24 @@ public function __construct()
             'sessions/{sessionId}' => function (array $content) {
                 return new Session($content);
             },
-            'login' => function (array $content) {
+            'users/login' => function (array $content) {
                 return new Session($content);
             },
+            'users/signup' => function (array $content) {
```

Comment:
Do we really have this endpoint? AFAIU, this is to make `POST` to `users/signup` w/ merchant data and create it. So after `POST` we redirect to `merchants/{id}`.

Also, why we do not use standard `POST /merchants/` endpoint ?